ractor_cluster/src/net/listener.rs

```diff
@@ -110,7 +110,10 @@ impl Actor for Listener {
                                     stream: enc_stream,
                                 }),
                                 Err(some_err) => {
-                                    log::warn!("Error establishing secure socket: {}", some_err);
+                                    tracing::warn!(
+                                        "Error establishing secure socket: {}",
+                                        some_err
+                                    );
```

Comment:
Nit: use the `{some_err}` syntax? Or is there an MSRV constraint? Or just don't like it?

Reply:
Hmm I thought clippy caught these, this is just a general oversight. Will fix.

Reply:
Hmm I thought clippy caught these, this is just a general oversight. Ah perhaps my local clippy is out of date, will re-run or fix manually. thanks

Reply:
https://github.com/slawlor/ractor/pull/134/commits/fa9faa2764262acc421c5b7ee4626acffd50e98e#diff-404312f7298c59a6ebaf75fdf76f00c44ecb4351fbe48e9d07d8bd71f1d57913